tests/visco_plastic_differentplasticitydifferentphases/screen-output

```diff
@@ -0,0 +1,76 @@
+
+
+
++----------------------------------------------+------------+------------+
+| Total wallclock time elapsed since start     |     0.377s |            |
+|                                              |            |            |
+| Section                          | no. calls |  wall time | % of total |
++----------------------------------+-----------+------------+------------+
+| Assemble Stokes system           |         1 |    0.0053s |       1.4% |
+| Assemble temperature system      |         1 |   0.00623s |       1.7% |
+| Build Stokes preconditioner      |         1 |    0.0135s |       3.6% |
+| Build temperature preconditioner |         1 |   0.00227s |       0.6% |
+| Initialization                   |         1 |     0.288s |        76% |
+| Postprocessing                   |         1 |    0.0297s |       7.9% |
+| Setup dof systems                |         1 |   0.00401s |       1.1% |
+| Setup initial conditions         |         1 |   0.00504s |       1.3% |
+| Setup matrices                   |         1 |   0.00473s |       1.3% |
+| Solve Stokes system              |         1 |   0.00495s |       1.3% |
+| Solve temperature system         |         1 |  0.000883s |      0.23% |
++----------------------------------+-----------+------------+------------+
+
+Termination requested by criterion: end time
+
+
++----------------------------------------------+------------+------------+
+| Total wallclock time elapsed since start     |     0.377s |            |
+|                                              |            |            |
+| Section                          | no. calls |  wall time | % of total |
++----------------------------------+-----------+------------+------------+
+| Assemble Stokes system           |         1 |    0.0053s |       1.4% |
+| Assemble temperature system      |         1 |   0.00623s |       1.7% |
+| Build Stokes preconditioner      |         1 |    0.0135s |       3.6% |
+| Build temperature preconditioner |         1 |   0.00227s |       0.6% |
+| Initialization                   |         1 |     0.288s |        76% |
+| Postprocessing                   |         1 |    0.0297s |       7.9% |
+| Setup dof systems                |         1 |   0.00401s |       1.1% |
+| Setup initial conditions         |         1 |   0.00504s |       1.3% |
+| Setup matrices                   |         1 |   0.00473s |       1.3% |
+| Solve Stokes system              |         1 |   0.00495s |       1.3% |
+| Solve temperature system         |         1 |  0.000883s |      0.23% |
++----------------------------------+-----------+------------+------------+
+
+-----------------------------------------------------------------------------
+-- For information on how to cite ASPECT, see:
+--   https://aspect.geodynamics.org/citing.html?ver=2.4.0-pre&sha=a8131d2&src=code
+-----------------------------------------------------------------------------
```

Comment:
It's usually easiest to move and rename the file generated by ctest (the one with the suffix .notime), which doesn't have all the version and timing information. Or you can manually remove all lines beginning `--`, `|` and `+`.